.github/workflows/main.yml

```diff
@@ -78,7 +78,7 @@ jobs:
             os: windows-latest
             tox_env: "py39-xdist"
           - name: "windows-py310"
-            python: "3.10-dev"
+            python: "3.10.1"
```

Comment:
Note: for now it seems `3.10.1` is not the default image if we just leave it as `"3.10"` (it installs `3.10.0` in that case).

We probably want to change this to `"3.10"` once it installs `3.10.1` (or later).